libcontainerd/client_windows.go

```diff
@@ -305,21 +320,21 @@ func (clnt *client) createLinux(containerID string, checkpoint string, checkpoin
 		},
 	}
 
-	var layerOpt *LayerOption
-	for _, option := range options {
-		if l, ok := option.(*LayerOption); ok {
-			layerOpt = l
-		}
+	if spec.Windows == nil {
+		return fmt.Errorf("spec.Windows must not be nil for LCOW containers")
 	}
 
-	// We must have a layer option with at least one path
-	if layerOpt == nil || layerOpt.LayerPaths == nil {
-		return fmt.Errorf("no layer option or paths were supplied to the runtime")
+	// We must have least two layers in the spec, the bottom one being a base image,
+	// the top one being the RW layer.
```

Comment:
That is correct, it was. But two is correct (for Windows containers, not lcow) in the option-free mode. We combined a field into the layer folders